### PR TITLE
Fix pkg.go.dev link for certificates README

### PR DESCRIPTION
**Pull request by kimprice**
The link right now is going to the Python library. Also fixed a couple errors in the sample code.

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

File: sdk/keyvault/azcertificates/README.md [hunk 1]
```diff
@@ -450,12 +450,12 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [azure_sub]: https://azure.microsoft.com/free/
 [code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
 [keyvault_docs]: https://docs.microsoft.com/azure/key-vault/
-[pkggodev_azcerts]: https://pypi.org/project/azure-keyvault-certificates/
+[pkggodev_azcerts]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/keyvault/azcertificates
 [certificate_client_docs]: https://aka.ms/azsdk/go/azcertificates
 [rbac_guide]: https://docs.microsoft.com/azure/key-vault/general/rbac-guide
 [reference_docs]: https://aka.ms/azsdk/go/azcertificates
```
Comment (kimprice) [inline sdk/keyvault/azcertificates/README.md:456]: I'm also wondering about the reference docs link. In some of our other READMEs the API Reference is a link to the pkg.go.dev, so it seems redundant here.

**Comment by kimprice** [comment]
I wonder if we should also have a general README at the root for KeyVault with just basic information on KeyVault like what's at the beginning of this README. Then in the READMEs for the individual package, they would just focus on that package and the service docs linked could be for that specific Key Vault management.

**Review by seankane-msft — APPROVED** [review]

**Comment by seankane-msft** [comment]
> I wonder if we should also have a general README at the root for KeyVault with just basic information on KeyVault like what's at the beginning of this README. Then in the READMEs for the individual package, they would just focus on that package and the service docs linked could be for that specific Key Vault management.

I checked the other language repos, they don't do this but I think it could be worth leading the charge here.

**Comment by kimprice** [comment]
> > I wonder if we should also have a general README at the root for KeyVault with just basic information on KeyVault like what's at the beginning of this README. Then in the READMEs for the individual package, they would just focus on that package and the service docs linked could be for that specific Key Vault management.
> 
> I checked the other language repos, they don't do this but I think it could be worth leading the charge here.

I guess the only downside is that it wouldn't technically be part of a package. So maybe what we need to do is have a consistent intro section for all the Key Vault libraries that starts by introducing Key Vault and describes the three separate packages (exactly like what's in this README).

**Pull request merged**